### PR TITLE
Add support for multiple account owners in Unit

### PR DESCRIPTION
**Pull request by rahul-x-verma**

### CODE REVIEW

File: packages/sandbar/src/client.ts [hunk 1]
```diff
@@ -154,6 +168,19 @@ export class Client {
     }
   }
 
+  convertUnitRelationships<T> (unitObject: CorrectUnitRelationships<T>) : any | undefined {
```
Comment (rahul-x-verma) [inline packages/sandbar/src/client.ts]: Is there a better return type than "any"? I tried "T", but that didn't compile.

**Comment by rahul-x-verma** [comment]
Verified locally by modifying the unit-example library to use the new endpoints.

File: packages/sandbar/src/client.ts (hunk 1 above)
Comment (ryanrhee) [reply packages/sandbar/src/client.ts]: definitely a better type. please provide more details on how / why using `T` did not compile.

**Review by ryanrhee — CHANGES_REQUESTED** [review]
also, jest is failing

File: packages/sandbar/src/client.ts [hunk 2]
```diff
@@ -154,6 +169,24 @@ export class Client {
     }
   }
 
+  convertUnitRelationships<T>(
```
Comment (ryanrhee) [inline packages/sandbar/src/client.ts:172]: move this out of the class

**Review by ryanrhee — APPROVED** [review]

**Pull request merged**